include/z64actor.h

```diff
@@ -132,6 +132,20 @@ typedef struct {
     /* 0x18 */ Vec3f feetPos[2]; // Update by using `Actor_SetFeetPos` in PostLimbDrawOpa
 } ActorShape; // size = 0x30
 
+#define BGCHECKFLAG_GROUND (1 << 0) // Standing on the ground
+#define BGCHECKFLAG_GROUND_TOUCH (1 << 1) // Has touched the ground (only active for 1 frame)
+#define BGCHECKFLAG_GROUND_LEAVE (1 << 2) // Has left the ground (only active for 1 frame)
+#define BGCHECKFLAG_WALL (1 << 3) // Touching a wall
+#define BGCHECKFLAG_CEILING (1 << 4) // Touching a ceiling
+#define BGCHECKFLAG_WATER (1 << 5) // In water
+#define BGCHECKFLAG_WATER_TOUCH (1 << 6) // Has touched water (reset when leaving water)
+#define BGCHECKFLAG_GROUND_STRICT (1 << 7) // Strictly on ground (BGCHECKFLAG_GROUND has some leeway)
```

Comment:
Can the comment here (or for `BGCHECKFLAG_GROUND`) elaborate on what this "leeway" is? I see in the removed docs that this flag is `Similar to & 0x1 but with no velocity check and is cleared every frame`. Is that the difference?

Reply:
I made a pastebin with rough docs of the relevant function:
https://pastebin.com/Mc2c9m3N

There was slightly more distinction in OoT due to the slightly above ground check (`< 11.0f`) being in `Actor_BgCheckLeaveGround`, which gave leeway to `BGCHECKFLAG_GROUND` being set even when the actor was single unit digits above the floor, while `BGCHECKFLAG_GROUND_STRICT` couldn't have that gap.

But in MM, it's been moved to the main actor bgcheck ground function, meaning `BGCHECKFLAG_GROUND` and `BGCHECKFLAG_GROUND_STRICT` seem to be more in sync with one another.

I think the original comment `Similar to & 0x1 but with no velocity check and is cleared every frame` is better here and will revert.